### PR TITLE
GH-1347 GOOGLE_COMBO_COMPAT

### PR DESCRIPTION
**Pull request by mlzummo**
Fixes GH-1347

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @jbrundage please review

File: test/googleFactory.js [hunk 1]
```diff
@@ -54,7 +54,9 @@
                     callback(new Combo()
                         .columns(DataFactory.ND.fivecolumn.columns)
                         .data(DataFactory.ND.fivecolumn.data)
-                        .customSeries({2: {type: 'line'}})
+                        // .columns(DataFactory.ND.subjects.columns)
+                        // .data(DataFactory.ND.subjects.data)
```
Comment (GordonSmith) [inline test/googleFactory.js]: remove commented code.

**Comment by mlzummo** [comment]
@GordonSmith fixed

File: test/amchartFactory.js [hunk 2]
```diff
@@ -14,22 +14,22 @@
...
-                            "value": new Date("2014-07-13"),
-                            "toValue": new Date("2014-08-13"),
-                            "lineAlpha": 0.2,
-                            "lineColor": "#00cc00",
-                            "lineThickness": 3,
-                            "fillAlpha": 0.1,
-                            "fillColor": "#00cc00",
-                            "label": "Guide #1",
-                            "inside": false
+                            "value": new Date("2014-06-13"),
```
Comment (jbrundage) [inline test/amchartFactory.js]: @mlzummo Revert the changes to this file
